docs/concepts/copilot/copilot-data-privacy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Data Copilot data privacy
```

Comment:
The word data is used twice...maybe emphasize it like this?
"Data Privacy with Data Copilot"

Reply:
Done.